apps/teams-test-app/e2e-test-data/webStorage.json

```diff
@@ -7,15 +7,39 @@
   },
   "checkIsSupported": {
     "domElementName": "checkWebStorageCapability",
-    "expectedOutput": "webStorage is not supported"
+    "expectedOutput": "webStorage is supported"
   },
   "testCases": [
     {
       "title": "isWebStorageClearedOnUserLogOut function Call - Success",
       "type": "callResponse",
       "boxSelector": "#box_isWebStorageClearedOnUserLogOut",
       "expectedAlertValue": "isWebStorageClearedOnUserLogOut called",
-      "expectedTestAppValue": "webStorage is cleared on user log out"
+      "expectedTestAppValue": "webStorage is cleared on user log out",
+      "hostSdkVersion": {
+        "web": "<4.0.2"
```

Comment:
I think this PR might need to wait for a little bit since Web Host SDK is releasing and I am not sure which version you should exactly to have here.

Reply:
sounds good, will wait